app/src/lib/constants.ts

```diff
@@ -1,12 +1,14 @@
 export const PAGE_SIZE = 50;
 
+// TODO: get this from api and store in local storage
```

Comment:
```suggestion
// TODO: get this from api
```

there's really no point in keeping this in local storage